### PR TITLE
disabling new member functionality

### PR DESCRIPTION
**Pull request by Vimalnegi03**
Targets #1624

### Description
I have successfully disabled the add new member functionality for  inactive or halted members.. 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php [hunk 1]
```diff
@@ -94,11 +94,15 @@
                         </div>
                     </div>
                 </div>
-
+
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php:97]: Remove extra space.

File: Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php [hunk 2]
```diff
@@ -94,11 +94,15 @@
...
+            
                 <div>
+                    
+                    
+                    @if($project -> status == 'active')
                     <span v-on:click="addNewProjectTeamMember()" style="text-decoration: underline;" class="text-underline btn" >Add new team member</span>
+               @endif
                 </div>
-
+
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php:105]: Same here. Remove extra line.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

File: Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php (hunk 2 above)
Comment (Vimalnegi03) [reply Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php]: okay sir

File: Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php [hunk 3]
```diff
@@ -96,11 +96,13 @@
                 </div>
 
                 <div>
-                    <span v-on:click="addNewProjectTeamMember()" style="text-decoration: underline;" class="text-underline btn" >Add new team member</span>
-                </div>
 
+                    @if($project -> status == 'active')
+                    <span v-on:click="addNewProjectTeamMember()" style="text-decoration: underline;" class="text-underline btn" >Add new team member</span>
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php:101]: Please use bootstrap class instead of inline CSS.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

**Review by Ayush8923 — APPROVED** [review]
LGTM👍

**Pull request merged**